### DISCUSSION TITLE
Where is the source for the API docs

### DISCUSSION THREAD

**Discussion by agent00jackson** [reactiveui/ReactiveUI#3104] [General]
Hi! The API docs look like docfx, I was curious how the docfx project was set up. Mainly interested in how the mermaid diagrams are done.

**Comment by glennawatson** [comment]
Uses a project called wyam to generate. Which is being replaced in the future by statiq.

**Reply by agent00jackson** [reply]
Cool, thanks! Didn't see that it was generating the diagrams at first but I see it now.
